content-node/docker-compose/setup.sh

```diff
@@ -141,6 +141,23 @@ get_validated_input() {
     done
 }
 
+# Check if .env file exists and prompt for confirmation
+if [ -f ".env" ]; then
+    log_warning "An .env file already exists!"
```

Comment:
should we print values here?

Reply:
Yeah, there are no secrets, we can log this.